lib/xcode.js

```diff
@@ -109,7 +109,8 @@ async function getVersionWithoutRetry () {
 
   // we want to read the CFBundleShortVersionString from Xcode's plist.
   // It should be in /[root]/XCode.app/Contents/
-  const plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+  let plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+  plistPath = plistPath.replace(new RegExp("/$"), "").trim();
```

Comment:
I'd like you to execute the following:

```
var xcodePath = '/Applications/Xcode.app/Contents/Developer/';
var plistPath = xcodePath.replace("/Contents/Developer",  "/Contents/Info.plist");
```

then tell me what the value of plistPath is.